### PR TITLE
Updated and fixed IntelliJ quickstart

### PR DESCRIPTION
**Pull request by Neurone**
**Description**:

This PR modifies and fixes the IntelliJ quickstart guide
* Updated JDK 17 (text and images)
* Fixed config.txt
* Fixed dependency (besu-internal-version)

**Related issue(s)**:

N/A

**Notes for reviewer**:

N/A

**Checklist**

- [x] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: docs/intellij-quickstart.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # IntelliJ quickstart
 
 ## JVM
-OpenJDK12 is strongly recommended. You can [download it from IntelliJ](https://www.jetbrains.com/help/idea/sdk.html)
+OpenJDK 17 is strongly recommended. You can [download it from IntelliJ](https://www.jetbrains.com/help/idea/sdk.html)
```
Comment (nathanklick) [inline docs/intellij-quickstart.md:4]: OpenJDK 17 is now mandatory as of mainnet release `v0.26.x`. I would suggest changing this to say `required` or similar verbiage.

Comment (nathanklick) [reply docs/intellij-quickstart.md:4]: Additionally, I would suggest updating the SDK download URL to link to the Adoptium builds.

**Review by nathanklick — COMMENTED** [review]
Might also want to include steps for importing the gradle project since the Maven support will be deprecated soon and eventually removed in the `v0.29.0` releases.

**Comment by Neurone** [comment]
Sure, I can do it.

A question: do we use HTML syntax in this MD file for some specific reason? If not, can I change those tags with the corresponding MD syntax?

File: docs/intellij-quickstart.md (hunk 1 above)
Comment (Neurone) [reply docs/intellij-quickstart.md:4]: > OpenJDK 17 is now mandatory as of mainnet release `v0.26.x`. I would suggest changing this to say `required` or similar verbiage.

Ok, we should update the root [README](https://github.com/hashgraph/hedera-services/blob/master/README.md#jvm) also

**Comment by Neurone** [comment]
@nathanklick I reviewed the doc, it should be fine now.

**Pull request closed without merging**